packages/@lwc/engine-core/src/framework/hot-swaps.ts

```diff
@@ -25,13 +25,13 @@ const activeStyles = new WeakMap<StylesheetFactory, Set<VM>>();
 function rehydrateHotTemplate(tpl: Template): boolean {
     const list = activeTemplates.get(tpl);
     if (!isUndefined(list)) {
-        list.forEach((vm) => {
+        for (const vm of list) {
```

Comment:
Using `forEach.call` is pretty risky, because you can accidentally use it on a `Map` or `Set` and TypeScript won't catch it. (Found this through a Karma test failure.) So I'm preferring `for...of` instead.